rfml_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"testing"
 
 	"github.com/rainforestapp/rainforest-cli/rainforest"
+	"github.com/urfave/cli"
```

Comment:
is this used for anything?

Reply:
Go won't let you include things that are not used, it raises an error at compile time!
e.g. https://play.golang.org/p/ndFKKsMopT